stable/grafana/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: grafana
-version: 5.0.13
+version: 5.0.14
```

Comment:
Could you bump to 5.0.16 please? @blagojts

Reply:
@zanhsieh bumbed to 5.0.16